contrib/Detached.Mappers.EntityFramework.Contrib.SysTec/ComplexDbContext.cs

```diff
@@ -51,6 +52,13 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
                     //    .Value(nameof(Customer), typeof(Customer))
                     //    .Value(nameof(Government), typeof(Government))
                     //    .Value(nameof(SubGovernment), typeof(GovernmentLeader));
+
+                    options.Type<OrganizationBaseDTO>().Abstract();
```

Comment:
abstract types are dynamic in runtime, that means, the mapper checks for the object type at mapping time
normal types are static, that means, the mapper is defined before you call to the Map operation
for test 13 to work, we force Abstract on OrgaizationBaseDTO, even when the class is not marked as abstract

Reply:
OK, if a type is abstract you search for inheritance (which isn't completely correct, but I can live with that).

OrganizationBaseDTO should be public abstract OrganizationBaseDTO. If it is not possible or wrong designed, you can do the trick with your own marker Abstract().